Moli/DataModels/ProfileImagesDataModel/ProfileImagesDataModel.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import SwiftUI
+import Mantis
+
+class ProfileImagesDataModel: ObservableObject {
+    @Published var imageType: ImageType = .profile
+    @Published var shouldShowPhotoConfirmDialog = false
+    @Published var sourceType: UIImagePickerController.SourceType = .photoLibrary
+    @Published var isImagePickerDisplay = false
+
+    @Published var uiImageToCrop: UIImage = UIImage(named: "ProfilePlaceHolder")!
```

Comment:
:question: Som jag har förstått från Design så kommer vi inte köra på någon default image for now (vi kör initialer om man inte har någon bild), så det här värdet kanske ska vara en optional isåfall? Så får man hantera med hjälp av guard dess värde osv
```suggestion
    @Published var uiImageToCrop: UIImage?
```

Reply:
Problemet som jag stötte på när jag försökte med det var att Imagepicker eller imagecropper protokollet inte tillät det. Därför lät jag det vara så med profile placeholder